itest/data/srv-configs/backend.main.yaml

```diff
@@ -39,6 +39,22 @@ cached_endpoints:
         id_identifier: 'id'
         pattern: "(^/not_authorized\\?ids=)((?:\\d|%2C)+)(.*$)"
         ttl: 30
+    # POST endpoints cached.
+    post_no_id:
+        ttl: 30
+        pattern: '^/post_always_cache/$'
+    post_with_id:
+        ttl: 30
+        pattern: '^/post_id_cache/$'
+        enable_id_extraction: true
+        post_body_id: request_id
+    post_with_id_varying_body:
+        ttl: 30
+        pattern: '^/post_id_cache_variable_body/$'
+        enable_id_extraction: true
+        post_body_id: request_id
+        vary_body_field_list: ['ignore_field1', 'ignore_field2']
```

Comment:
In case of GETs, vary_headers is a whitelist. Can we make it so also here?

The reason why they're called varying is because different values for these headers will cause a different response. The other headers (i.e. user-agent) are ignored and the response will be the same whatever their value.

Reply:
done